apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/actions.ts

```diff
@@ -11,22 +11,33 @@ import { getSurvey, updateSurvey } from "@formbricks/lib/survey/service";
 import { formatSurveyDateFields } from "@formbricks/lib/survey/util";
 import { AuthenticationError, AuthorizationError, ResourceNotFoundError } from "@formbricks/types/errors";
 
-type TSendEmailActionArgs = {
-  to: string;
-  subject: string;
-  html: string;
-};
-
-export const sendEmailAction = async ({ html, subject, to }: TSendEmailActionArgs) => {
+export const sendEmbedSurveyPreviewEmailAction = async ({ surveyId }: { surveyId: string }) => {
```

Comment:
can we also pass the arguments directly without an object. This is how we do it in most actions and since an action is a function and not a component I think this makes sense :-)